src/GeoInterface.jl

```diff
@@ -1,6 +1,7 @@
 module GeoInterface
 
-using Extents
+import Extents
+using Extents: Extent
```

Comment:
I usually combine the two like this in one line:

```suggestion
using Extents: Extents, Extent
```